doc/lightning_qubit/development/avx_kernels/build_system.rst

```diff
@@ -8,11 +8,10 @@ when the library is loaded, and it is used at runtime when it is the most suitab
 To support AVX2 and AVX512 kernels, we always compile those kernels if the target system is UNIX on x86-64.
 Specifically, we made separate C++ files for AVX2 and AVX512 kernels and build them as a static library with the corresponding compile options. This is handled by CMake. One can check ``gates/CMakeLists.txt`` file for details.
 
-One caveat is that we want to make sure that default kernels (``KernelType::PI`` and ``KernelType::LM``) are only instantiated once with specific compiler flags during the compile process.
+One caveat is that we want to make sure that default ``KernelType::LM`` kernels are only instantiated once with specific compiler flags during the compile process.
 This is important as the linker sometimes cannot choose the right instantiation when there are multiple instantiations of the same template class.
 This problem does not arise when all instantiations are compiled with the same options, but with the AVX2/512 kernels, we use different compile options for each translation unit. We solve this problem by adding explicit instantiation declarations in the header files for these kernels
-(:ref:`file_pennylane_lightning_core_src_simulators_lightning_qubit_gates_cpu_kernels_GateImplementationsLM.hpp` and
-:ref:`file_pennylane_lightning_core_src_simulators_lightning_qubit_gates_cpu_kernels_GateImplementationsPI.hpp`)
+(:ref:`file_pennylane_lightning_core_src_simulators_lightning_qubit_gates_cpu_kernels_GateImplementationsLM.hpp`)
```

Comment:
If you click the link, the website does not render correctly. 
![image](https://github.com/user-attachments/assets/7c2708de-9ae4-42f2-9a0e-073f4f4e7836)

Reply:
This issue has been there as long as I could traverse over the history of API docs :(

Reply:
This issue has existed for as long as I can trace back through the history of the API docs :(